.woodpecker.yml

```diff
@@ -241,17 +241,29 @@ steps:
       - event: push
         branch: main
 
+  publish_builder_arm64:
+    image: woodpeckerci/plugin-docker-buildx
+    secrets: [docker_username, docker_password]
+    settings:
+      repo: dessalines/lemmy-builder-arm64
+      dockerfile: docker/builders/lemmy-builder-arm64
```

Comment:
I fucked up this line.. my bad!

Reply:
No probs, just ping me when its fixed and you need me to start another build.

Reply:
Done @dessalines , I also rebased on main.

Reply:
Thx. In the future do a `git merge origin/main`, because rebasing makes collaborating more difficult as it rewrites history.

Reply:
No probs. Build is started. Sometimes our builders have issues, so I might have to restart if it fails, to verify that its not any issue with your changes.

Reply:
No problem :) I started contributing to WP as well (https://github.com/woodpecker-ci/autoscaler/pull/56) so maybe you have some issue you would like me to work on that would benefit both project :D